docs/source/parallelism.mdx

```diff
@@ -308,8 +315,10 @@ And since we have ZeRO, the other benefit is ZeRO-Offload. Since this is stage 1
 
 Implementations:
 - [Megatron-DeepSpeed](https://github.com/microsoft/Megatron-DeepSpeed)
+- [OSLO](https://github.com/tunib-ai/oslo)
 
-🤗 Transformers status: not yet implemented, since we have no PP and TP.
+🤗 Transformers status:
+- [OSLO](https://github.com/tunib-ai/oslo) is under discussion about integration with Transformers.
```

Comment:
OSLO and Megatron-DeepSpeed both provide ZeRO + TP + PP without an internal ZeRO implementation, so I think they can be written in the same section.

Reply:
reverted.